sklearn/tests/test_discriminant_analysis.py

```diff
@@ -473,6 +568,68 @@ def test_qda_regularization():
     assert_array_equal(y_pred5, y5)
 
 
+def test_qda_ledoitwolf():
+    # test bad solver
+    clf = QuadraticDiscriminantAnalysis(
+            solver="svd",
+            covariance_estimator=LedoitWolf()
+    )
+    assert_raises(NotImplementedError, clf.fit, X, y)
```

Comment:
in our tests, we prefer to use the `with pytest.raises(NotImplementedError, match='...'):` construct.